tests/acceptance/features/apiFullTextSearch/search.feature

```diff
@@ -104,3 +104,30 @@ Feature: full text search
       | old              |
       | new              |
       | spaces           |
+
+
+  Scenario Outline: search project space files by content
+    Given using spaces DAV path
+    And the administrator has assigned the role "Space Admin" to user "Alice" using the Graph API
+    And user "Alice" has created a space "project-space" with the default quota using the GraphApi
+    And user "Alice" has created a folder "spacesFolderWithFile/spacesSubFolder" in space "project-space"
+    And user "Alice" has uploaded a file inside space "project-space" with content "hello world from nepal" to "keywordAtStart.txt"
+    And user "Alice" has uploaded a file inside space "project-space" with content "saying hello to the world" to "spacesFolderWithFile/keywordAtMiddle.txt"
+    And user "Alice" has uploaded a file inside space "project-space" with content "nepal want to say hello" to "spacesFolderWithFile/spacesSubFolder/keywordAtLast.txt"
+    And user "Alice" has uploaded a file inside space "project-space" with content "namaste from nepal" to "hello.txt"
+    And using <dav-path-version> DAV path
+    When user "Alice" searches for "Content:hello" using the WebDAV API
+    Then the HTTP status code should be "207"
+    And the search result of user "Alice" should contain only these files:
+      | keywordAtStart.txt  |
+      | keywordAtMiddle.txt |
+      | keywordAtLast.txt   |
+    Examples:
+      | dav-path-version |
+      | old              |
+      | new              |
+
+    @skipOnStable3.0
+    Examples:
+      | dav-path-version |
+      | spaces           |
```

Comment:
Please update this is master as well

Reply:
I have to update in other scenarios also. 
So, is it okay to update after all the scenario has concluded?